pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -58,6 +60,29 @@ def __enter__(self):
         print(msg)
         bokchoy_utils.check_services()
 
+        if not self.testsonly:
+            self.prepare_bokchoy_run()
+
+        msg = colorize('green', "Waiting for servers to start...")
+        print msg
+        bokchoy_utils.wait_for_test_servers()
+        if self.serversonly:
+            self.interruptable_prompt()
+
+    def __exit__(self, exc_type, exc_value, traceback):
```

Comment:
I left this in here since we'd want to know if the servers got killed (or the user forgot to start them). Otherwise finding out would likely happen with some confusing browser exceptions. That said, I've changed the wording so this waiting period is less confusing.